comp-sci/data-structures-and-algorithms/intro-data-structures/the-array-data-structure.md

```diff
@@ -28,7 +28,7 @@ parent: what-is-a-data-structure
 
 An **array** is a data structure organized as a linear collection of elements, for which an **index** (a numerical indicator of position) is used to access its content. The same information may appear in the array more than once but under a different index. This is how you could picture it:
 
-![array](https://img.enkipro.com/630c4678f67993a0479b25467cda6db9.png)
+![array](https://img.enkipro.com/0e755de4b3d32ffb01a139a3904e8d2d.png)
```

Comment:
here, the background of the squares shouldn't be orange

Reply:
They were colored when they were made but on the 2nd version(the one before this pr) they aren't. So I wasn't sure which version to do. The colors of the original versions are exactly as in this PR,

Reply:
Should I do them without the coloring then?